.gitignore

```diff
@@ -358,3 +358,6 @@ MigrationBackup/
 
 # Fody - auto-generated XML schema
 FodyWeavers.xsd
+
+# IntelliJ
+.idea/
```

Comment:
For reference:
https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-
https://github.com/JetBrains/resharper-rider-samples/blob/master/.gitignore